completion/available/gradle.completion.bash

```diff
@@ -22,7 +22,8 @@
 
 # Bash breaks words on : by default. Subproject tasks have ':'
 # Avoid inaccurate completions for subproject tasks
-COMP_WORDBREAKS=$(echo "$COMP_WORDBREAKS" | sed -e 's/://g')
+# shellcheck disable=SC2001
+COMP_WORDBREAKS="$(echo "$COMP_WORDBREAKS" | sed -e 's/://g')"
```

Comment:
I think we can just follow ShellCheck's suggestion.

```suggestion
COMP_WORDBREAKS=${COMP_WORDBREAKS//:}
```

Reply:
here I was not sure, as there may be more than one `:` which the /g will catch but parameter replacement won't, so I allowed myself to bet this is intentional.

Reply:
> which the /g will catch but parameter replacement won't,

The replacement with the parameter expansion does that with double `//`. I.e., `${COMP_WORDBREAKS/:}` to remove the first occurrence of `:`, and `${COMP_WORDBREAKS//:}` to remove all the occurrences of `:`.